### ISSUE TITLE
Compressing quality scores alone

### ISSUE THREAD

**Issue by sagnikbanerjee15** [Yoniming/code#2]
Hello,

I wish to compress a file that contains only quality scores. Can ACO perform lossless compression and decompression on such a file?

Thank you.

**Comment by Yoniming** [comment]
Yes, ACO is a compressor dedicated to compressing quality scores, you can do this by changing the file read and write. It should be noted that the context model used should also be modified appropriately, and the context model containing bases can no longer be used.
